user-guide/Standard_Apps/SRM/Interacting_with_SRM_using_code/SRM_silent_actions/SRM_creating_booking_silently.md

```diff
@@ -107,3 +107,40 @@ public class Script
 >
 > - In case a null service definition is provided, SRM will use a dynamically generated blank service definition.<!-- RN 30324 -->
 > - To add a resource without linking it to a node in the service definition, do not provide the *Id* attribute of the *Function* object.<!-- RN 30324 -->
+
+# Handling bookings with start date in the past
+
+When a booking is created with the method above and the start date is in the past is recomended to wait until the booking is fully started up before doing any other operation, otherwise some updates done by the start action may be overridden.
+
+The following code can be used to wait:
```

Comment:
S: ... to wait after *CreateNewBooking* successfully created the booking:

Reply:
done